### PR TITLE
Kathleen database settings page

### PR DESCRIPTION
**Pull request by cheshirekate8**
1. Wrote databaseSettingsTests/index
2. Deleted repoLeftNav
3. Renamed repoDatabaseNav to repoLeftNav

### CODE REVIEW

File: cypress/integration/productionTests/privatePaths/render/databaseSettingsPage/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import { macbook15ForAppLayout } from "../../../../utils/devices";
+import {
+  newClickFlow,
+  newExpectation,
+  newShouldArgs,
+  newExpectationWithClickFlows,
+  newExpectationWithScrollIntoView,
+} from "../../../../utils/helpers";
+
+const pageName = "Logged in database page with no branch and no data";
```
Comment (tbantle22) [inline cypress/integration/productionTests/privatePaths/render/databaseSettingsPage/index.spec.ts:11]: This should be "Database settings page"

File: cypress/integration/productionTests/publicPaths/render/database/emptyWithBranch.spec.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ import {
   testViewsSection,
   testQueryCatalogSection,
   testSchemaSection,
-} from "../../../../utils/sharedTests/repoDatabaseNav";
+} from "../../../../utils/sharedTests/repoLeftNav";
 import { testRepoHeaderWithBranch } from "../../../../utils/sharedTests/repoHeaderNav";
 // import { testSqlConsole } from "../../../../utils/sharedTests/sqlEditor";
```
Comment (tbantle22) [inline cypress/integration/productionTests/publicPaths/render/database/emptyWithBranch.spec.ts:17]: This should probably be removed?

**Review by tbantle22 — APPROVED** [review]
Awesome job! That's for cleaning up those sharedTests files

File: cypress/integration/productionTests/publicPaths/render/database/emptyWithBranch.spec.ts (hunk 2 above)
Comment (cheshirekate8) [reply cypress/integration/productionTests/publicPaths/render/database/emptyWithBranch.spec.ts:17]: Line 17 yeah? Like 15 is needed to import the tests

**Comment by tbantle22** [comment]
The commented out sql console import

**Pull request merged**